packages/cli/package.json

```diff
@@ -26,6 +26,7 @@
     "docs": "yarn oclif-dev readme --multi --dir=../docs/command-line-interface && yarn prettier ../docs/command-line-interface/*.md --write",
     "lint": "tslint -c tslint.json --project tsconfig.json",
     "prepublishOnly": "yarn run build && oclif-dev manifest && oclif-dev readme",
+    "prepack": "yarn clean && yarn run prepublishOnly",
```

Comment:
From the [docs](https://docs.npmjs.com/misc/scripts): 
```
prepack: run BEFORE a tarball is packed (on npm pack, npm publish, and when installing git dependencies)
```

So running `prepublish` on `prepack` means running it twice when yo do `publish`

Maybe use `prepack` instead of `prepublishOnly`?
Not sure what it means by `installing git dependencies`. 
I would dig a bit deeper to undsertand what's the rigth script-hook to do the building

Reply:
> Maybe use prepack instead of prepublishOnly?

Yeah, sounds good.

> Not sure what it means by installing git dependencies.

I believe it means when someone adds a dependency of the form `git+https://git@github.com/visionmedia/express.git` or `git+ssh://git@github.com/visionmedia/express.git`

`prepack` seems like the right behavior in that case too. If not. I can remove `prepack` and call `yarn run prepublishOnly` before `yarn pack` explicitly as well.